frontend/src/pages/Scans/ScansView.tsx

```diff
@@ -239,6 +239,35 @@ const ScansView: React.FC = () => {
         <br></br>
         <Button type="submit">Create Scan</Button>
       </Form>
+      <ImportExport<Scan>
+        name="scans"
+        fieldsToExport={['name', 'arguments', 'frequency']}
+        onImport={async results => {
+          // TODO: use a batch call here instead.
+          let createdScans = [];
+          for (let result of results) {
+            createdScans.push(
+              await apiPost('/scans/', {
+                body: {
+                  ...result,
+                  // These fields are initially parsed as strings, so they need
+                  // to be converted to objects.
+                  arguments: JSON.parse(
+                    ((result.arguments as unknown) as string) || ''
```

Comment:
@epicfaace Can we just do `as string` instead here?

Reply:
No, because the `arguments` type is an object (even though it's really a string).